app-misc/socnetv/socnetv-3.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+DESCRIPTION="Qt Social Network Visualizer"
+HOMEPAGE="https://socnetv.org/"
+SRC_URI="https://github.com/socnetv/app/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/app-${PV}"
+
+LICENSE="GPL-3"
```

Comment:
https://github.com/socnetv/app/blob/92f14110360bb5007cfbc3a22d802576eab9ba44/socnetv.spec#L40 says GPL-3+ but nothing else seems to. Maybe ask them for clarification?

Reply:
Seems it is explicitly GPL-3+ : 
https://github.com/socnetv/app/commit/97d200c0ec681e8de06e4c4330121536c525ef66